paste.c

```diff
@@ -8,7 +8,7 @@ int wmain() {
 	HANDLE clip = GetClipboardData(CF_UNICODETEXT);
 	if (clip) {
 		_setmode(_fileno(stdout), _O_U8TEXT);
-		wprintf(L"%s\n", clip);
+		wprintf(L"%s", clip);
```

Comment:
@sindresorhus @Qix- I guess then test is wrong – copy reads file from stdin without additional new line, but paste will add one and files will not match.